shared/middleware/channels_api.rb

```diff
@@ -300,6 +300,26 @@ class ChannelsApi < Sinatra::Base
     {abuse_score: value}.to_json
   end
 
+  #
+  # POST /v3/channels/<channel-id>/featured-abuse
+  #
+  # Set the abuse score to -1000 to prevent reported abuse from # hiding a project. Can only be done by a project validator
+  # and should only be used when a project is featured.
+  #
+  post %r{/v3/channels/([^/]+)/featured-abuse$} do |id|
+    # UserPermission::PROJECT_VALIDATOR
+    not_authorized unless project_validator?
+
+    dont_cache
+    content_type :json
+    begin
+      value = StorageApps.new(storage_id('user')).set_abuse(id, -1000)
+    rescue ArgumentError, OpenSSL::Cipher::CipherError
+      bad_request
+    end
+    {abuse_score: value}.to_json
+  end
```

Comment:
Why does channels needs this new `/featured-abuse` API, whereas assets/files appear to already have a path for updating?

If this is needed, I wonder if there's an argument for having the score be customizable by the client  (rather than hardcoded to -1000 here).